Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -148,31 +148,31 @@
     <value>Token response had neither token nor access_token.</value>
   </data>
   <data name="MissingBaseImageConfigNode" xml:space="preserve">
-    <value>Expected base image to have a config node</value>
+    <value>Expected base image to have a config node.</value>
   </data>
   <data name="MissingLayerNode" xml:space="preserve">
```

Comment:
I don't see any usage of MissingLayerNode in the project at all - maybe we can just remove this?

Reply:
The code was probably removed after creation of the branch :) Removed.